### PR TITLE
DRY the functions a bit; removed default bindings; add provides

### PR DESCRIPTION
**Pull request by ionrock**
Just some refactoring of the functions.

### CODE REVIEW

File: test-kitchen-mode.el [hunk 1]
```diff
@@ -56,10 +55,17 @@
 
 ;;; Code:
 
-(defvar kitchen-destroy-command "chef exec kitchen destroy")
-(defvar kitchen-list-command "chef exec kitchen list")
-(defvar kitchen-test-command "chef exec kitchen test")
-(defvar kitchen-verify-command "chef exec kitchen verify")
+(defcustom kitchen-destroy-command "chef exec kitchen destroy"
```
Comment (jjasghar) [inline test-kitchen-mode.el]: Shouldn't we put that `EXPORT=` you tweeted at me?

**Comment by jjasghar** [comment]
Also, it's obvious you're better at elisp then me. If you'd like to take a stab at: https://github.com/milkypostman/melpa/pull/2722 these changes that would be awesome.

File: test-kitchen-mode.el (hunk 1 above)
Comment (ionrock) [inline test-kitchen-mode.el]: Ah, you're right! I'll definitely add that.

**Comment by ionrock** [comment]
@jjasghar I update the PR with what I hope addresses the melpa issues. I also switched the README to org as it seems a bit more common amongst emacs packages. If that is something you can't stand, no worries. 

Let me know if I broke anything for you and thanks for writing this up!

**Pull request merged**